compose/seadoc-standalone.yml

```diff
@@ -0,0 +1,32 @@
+---
+services:
+
+  seadoc:
+    image: ${SEADOC_IMAGE:-seafileltd/sdoc-server:1.0-latest}
+    container_name: seadoc
+    volumes:
+      - ${SEADOC_VOLUME:-/opt/seadoc-data/}:/shared
+    # ports:
+    #   - "80:80"
+    environment:
+      - DB_HOST=${SEATABLE_MYSQL_DB_HOST:-mariadb}
+      - DB_PORT=${SEATABLE_MYSQL_DB_PORT:-3306}
+      - DB_USER=${SEATABLE_MYSQL_DB_USER:-root}
+      - DB_PASSWORD=${SEATABLE_MYSQL_ROOT_PASSWORD:?Variable is not set or empty}
+      - DB_NAME=${SEADOC_MYSQL_DB_NAME:-dtable_db}
+      - TIME_ZONE=${TIME_ZONE:-Etc/UTC}
+      - JWT_PRIVATE_KEY=${DTABLE_PRIVATE_KEY:?Variable is not set or empty}
+      - NON_ROOT=${NON_ROOT:-false}
+      - SEAHUB_SERVICE_URL=${SEATABLE_SERVER_PROTOCOL:-https}://${SEATABLE_SERVER_HOSTNAME:?Variable is not set or empty}
+    networks:
+      - frontend-net
+      - backend-seatable-net
+    labels:
+      caddy: ${SEADOC_SERVER_PROTOCOL:-https}://${SEADOC_SERVER_HOSTNAME:?Variable is not set or empty}
+      caddy.reverse_proxy: "{{upstreams 80}}"
+
```

Comment:
I would add a healthcheck similar to https://github.com/seatable/seatable-release/blob/a07cd29340874adb731f5943fea3c9b9ad792b7e/compose/seatable-server.yml#L87-L92